src/components/common/FormField.tsx

```diff
@@ -47,6 +50,7 @@ const FormField: React.FC<{
   readOnly?: boolean;
   disabled?: boolean;
   numeric?: boolean;
+  isMultiline?: boolean;
```

Comment:
nit: don't have to change it here, but at this point `FormField` is getting pretty clunky with a lot of props that are simply passed through to the material UI `TextField` component - instead of catching them explicitly and passing it through here, we should just take in additional props to `FormField` and spread them into `TextField`; this lets us catch only the props we care about in this component (which there are very few) and handle them accordingly while also allowing us to pass relevant props to `TextField` without worrying about them

again not necessary for this PR but if we need to add another prop to this component, we should consider this approach